lib/src/tabs/tab-set.component.ts

```diff
@@ -102,4 +102,20 @@ export class TabSetComponent implements AfterContentInit {
         }
         return routerLink;
     }
+
+    private routerMatchFound(routerLink: string): boolean {
```

Comment:
Just a thought, could probably make this func a little shorter, or even keep it inline (up on ln 89):
```
const currentRoute = this.router.url;
return currentRoute === routerLink || currentRoute.indexOf(`${routerLink}/`) > -1;
```